### DISCUSSION TITLE
adding a background image from the database using mysql and ejs

### DISCUSSION THREAD

**Discussion by Azzedine-zemmari** [community/community#133070] [Programming Help]
### Select Topic Area

Question

### Body

this is my code 
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Agenda</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Heebo&family=Roboto:wght@900&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="/public/css/Agenda.css">
</head>

<body>
    <div id="title" class="center">
        <h1>2024</h1>
        <p>Cycle Annuel de Croissance et de Développement des olives</p>
    </div>

    <div class="calendar-frame center">
        <div class="day-frame placeholder"></div>
        <div class="day-frame placeholder"></div>
        <% for(let i=0; i < data.length; i++) { %>
            <div class="day-frame center real" data-title="<%= data[i].title %>" data-time="<%= data[i].date %>"
                data-description="<%= data[i].description %>" style="background-image: url('<%= data[i].image %>'); background-size: cover; background-position: center;">
                <h2 class="day-number">
                    <%= data[i].id %>
                </h2>
                <div class="day-label center">
                    <%= data[i].title %>
                </div>
            </div>
            <% } %>
    </div>

    
    <div id="popup" class="popup">
        <div class="popup-content">
            <span class="close-btn">&times;</span>
            <p id="popup-time"></p>
            <h2 id="popup-title"></h2>
            <p id="popup-description"></p>
        </div>
    </div>

    <script src="/script.js"></script>
</body>

</html>
all the data came from the database only the background-image i think the syntax is the problem because ejs documentation not that specifique about that 
if you already been here help

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Azzedine-zemmari ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ebndev** [comment]
Hey @Azzedine-zemmari, thanks for being a part of the GitHub Community!

Does this div have a height + width in your CSS file? 

You can test if the image url is working by adding a new line: `<img src="<%= data[i].image %>">`

If both of those are working, then you can see if it is maybe a syntax issue.
